config/fpga/chan.yml

```diff
@@ -1,7 +1,7 @@
 dt: 0.1e-6
 func_order: 1
 func_numel: 512
-func_domain: [0.0, 3.19375e-9]
+func_domain: [0.0, 1.0e-9]
```

Comment:
Same comment as before; wanted to improve the accuracy of the step response.  Not sure if this was strictly necessary, though, since I ended up identifying other more significant causes of BER discrepancies.